ui/plugin-system/src/stories/shared-utils/decorators/WithDatasourceStore.tsx

```diff
@@ -31,7 +31,7 @@ function isWithDatastoreStoreParameter(
 export const WithDatasourceStore = (Story: StoryFn, context: StoryContext<unknown>) => {
   const { getPlugin } = usePluginRegistry();
 
-  const initParameter = context.parameters.WithDataQueries;
+  const initParameter = context.parameters.WithDatasourceStore;
```

Comment:
This was a copy-paste error that led to a really confusing experience when I was trying to set this parameter and it was conflicting with the other name. Fixing this while I'm in here.